src/store/token-swap/token-swap.reducer.js

```diff
@@ -0,0 +1,16 @@
+import { tokenSwapActionTypes } from './token-swap.actions'
+
+const initialTokenSwapState = {}
+
+function tokenSwapReducer (state = initialTokenSwapState, action) {
```

Comment:
Should we add the multistep process at this point to then make it easier to pick up a task for any of the other steps?

Reply:
I haven't added the steps here because I want to be sure first which data we will ned from the API and at which point in time. I would leave it as it's by the moment until we have the first step.